Lib/asyncio/streams.py

```diff
@@ -22,6 +23,26 @@
 _DEFAULT_LIMIT = 2 ** 16  # 64 KiB
 
 
+class StreamMode(enum.Enum):
+    READ = "read"
+    WRITE = "write"
+    READWRITE = "readwrite"
```

Comment:
I'd make this a Flag enum. Then you can use & and |.

Reply:
Nice idea!